lib/releases.sh

```diff
@@ -1,5 +1,30 @@
 #!/bin/bash
 
+# Requires parameters url and version. An optional parameter can be given to
+# exclude some versions. 
+# Example usage:
+# get_latest_release_from_goproxy "https://proxy.golang.org/sigs.k8s.io/cluster-api/@v/list" "v1.8." "beta|rc|pre|alpha"
+get_latest_release_from_goproxy() {
```

Comment:
It does not look like get_latest_release is used anywhere else, so you can probably replace the old function instead of creating a new one with a new name